api/src/main/java/org/openmrs/module/fgh/mpi/TokenInfo.java

```diff
@@ -0,0 +1,89 @@
+package org.openmrs.module.fgh.mpi;
+
+import org.apache.commons.lang.time.StopWatch;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class TokenInfo {
+	
+	@JsonProperty("access_token")
+	private String accessToken;
+	
+	@JsonProperty("id_token")
+	private String tokenId;
+	
+	@JsonProperty("token_type")
+	private String tokenType;
+	
+	@JsonProperty("expires_in")
+	private Double expiresIn;
+	
+	@JsonProperty("refresh_token")
+	private String refreshToken;
+	
+	private StopWatch stopWach;
```

Comment:
I thought we said don't use the stopwatch approach, it won't work correctly and hard to test, just capture the token expiry date and just compute the time elapsed